src/main/java/com/google/api/codegen/transformer/py/PythonImportSectionTransformer.java

```diff
@@ -14,20 +14,9 @@
  */
 package com.google.api.codegen.transformer.py;
 
-import com.google.api.codegen.config.ApiModel;
-import com.google.api.codegen.config.GapicProductConfig;
-import com.google.api.codegen.config.InterfaceModel;
-import com.google.api.codegen.config.ProductConfig;
-import com.google.api.codegen.config.TypeModel;
+import com.google.api.codegen.config.*;
```

Comment:
I believe star imports are discouraged. What prompted the change?

Reply:
Intellij handles my importing so /shrug. I'll look into it.